Gemfile

```
@@ -11,6 +11,7 @@ gem 'rspec', '~> 3.7'
 # RuboCop Performance upgrade to 1.11 is postponed until
 # https://github.com/rubocop/rubocop/pull/9721 will be resolved.
 gem 'rubocop-performance', '~> 1.10.0'
+gem 'rubocop-rake', '~> 0.5.1'
```

RuboCop Rake 0.6.0 has been released. Can you update to 0.6.0?
https://github.com/rubocop/rubocop-rake/releases/tag/v0.6.0

```suggestion
gem 'rubocop-rake', '~> 0.6.0'
```